src/uucore/src/lib/mods/error.rs

```diff
@@ -508,6 +508,17 @@ impl From<std::io::Error> for Box<dyn UError> {
     }
 }
 
+impl<T> FromIo<UResult<T>> for Result<T, nix::Error> {
```

Comment:
needs some comments here :)
and usage examples

Reply:
see the function below

Reply:
[c19c19e](https://github.com/uutils/coreutils/pull/4060/commits/c19c19e4db56678035d4aca4053128dfbcae7ed4)